### PR TITLE
Combining shamoon rohit branch changes

### PR DESCRIPTION
**Pull request by himanshudube97**
-> docker-compose.yml files had some conflicting thing. 
1- the architecture used- shamoon branch had apache/superset:3.1.0-py310-arm      (new everything machine supported that only.)
2-> port mapping was changed to  8082: 8088 (shamoon),  80:8088(Rohit),  -> (something else was running on port 80).
3-> container names changed.

### CODE REVIEW

**Comment by himanshudube97** [comment]
@fatchat  This is the branch which has the complete code
1. updated  jinja context
2. changes from yours and shamoon's branch.
3. my scripts and template files.

File: superset3/DockerFile.baseImage.template [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
+
+# Install Geckodriver
 RUN wget -q https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz && \
   tar -x geckodriver -zf geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz -O > /usr/bin/geckodriver && \
   chmod 755 /usr/bin/geckodriver && \
   rm geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz
 
-# python packages
-USER root
+# Upgrade pip
```
Comment (fatchat) [inline superset3/DockerFile.baseImage.template:20]: you removed `USER root`? does it still work?

Comment (himanshudube97) [reply superset3/DockerFile.baseImage.template:20]: > you removed `USER root`? does it still work?

yes, we already have USER root at the top level and we are only changin it back to superset user at the end, so it works.

**Comment by fatchat** [comment]
Closed by #52

**Pull request closed without merging**